client/components/payment-method-selector/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import FieldDescription from 'components/field-description';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import PaymentMethod from '../payment-method';
+import sanitizeHTML from 'lib/utils/sanitize-html';
+import { translate as __ } from 'lib/mixins/i18n';
+
+const PaymentMethodSelector = ( { paymentMethods, title, description, value, onChange } ) => {
+	const renderNoMethods = () => (
+		<p>
+			{ __( 'No payment methods available' ) }
+		</p>
+	);
+
+	const renderPaymentMethods = () => (
+		paymentMethods.map( ( paymentMethod ) => {
+			const methodValue = parseInt( paymentMethod.payment_method_id, 10 );
+			return (
+				<PaymentMethod
+					key={ paymentMethod.payment_method_id }
+					value={ methodValue }
+					currentValue={ value }
+					onChange={ () => onChange( methodValue ) }
+					name={ paymentMethod.name }
+					cardType={ paymentMethod.card_type }
+					cardDigits={ paymentMethod.card_digits }
+					expiry={ paymentMethod.expiry }
+				/>
+			);
+		} )
+	);
+
+	const hasMethods = 0 < paymentMethods.length;
+
+	return (
+		<FormFieldset className="payment-method-selector">
+			<FormLegend dangerouslySetInnerHTML={ sanitizeHTML( title ) } />
```

Comment:
Since this component is not data-driven, this `dangerouslySetInnerHTML` is not needed. If for some reason in the future we need to add mark-up to the title, we can just pass a React Component as the title instead of a string.